functorch/test/test_eager_transforms.py

```diff
@@ -2014,6 +2014,146 @@ def push_jvp(dummy, x):
         vmap(vmap(push_jvp, (0, None)))(dummy, x)
 
 
+# The tests here follow the cases in [Forward Grad View/inplace]
+# https://github.com/pytorch/pytorch/blob/master/torch/csrc/autograd/autograd_meta.cpp#L18-L43
+class TestVmapJvpInplaceView(TestCase):
+    # Case 1 in [Forward Grad View/inplace]
+    def test_all_dual_no_view(self, device):
+        B = 2
+
+        def push_jvp(f):
+            def inner(x, xt, y, yt):
+                return jvp(f, (x, y), (xt, yt))
+            return inner
```

Comment:
nit: maybe pull this and the version where x is not a dual into helper functions

Reply:
I didn't end up doing this because I didn't want to figure out the signature, but it's a good idea -- we want some sort of helper function that turns jvp into a higher-order function like our other transforms so that it can more easily be manipulated. Maybe that's just `functools.partial(jvp, f)`.